packages/venia-ui/lib/RootComponents/CMS/cms.js

```diff
@@ -35,7 +35,17 @@ const CMSPage = props => {
             data.cmsPage.content.length > 0 &&
             !data.cmsPage.content.includes('CMS homepage content goes here.')
         ) {
-            content = <RichContent html={data.cmsPage.content} />;
+            content = (
+                <>
+                    <Title>{data.cmsPage.title}</Title>
+                    {data.cmsPage.content_heading !== '' && (
+                        <h1 className="cms__content-heading">
```

Comment:
We don't have use global classnames in our components, and we don't target them with global CSS. If you'd like to render this element with some styles, consider defining a `.heading` classname in `cms.css`.

If styles are not being imported in this component yet, have a look at another component in the codebase to see how to import them, merge the classes, and apply them to elements. Something like the following:

```jsx
import defaultClasses from "./cms.css"

const CMSPage = props => {
  const classes = mergeClasses(defaultClasses, props.classes)

  return (
    <div className={classes.root} />
  )
}
```

```jsx
 
```

Reply:
No worries, can make this update